generators/app/templates/client/public/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title><%= _.kebabCase(appName) %></title>
+  </head>
+  <body>
+    <div id="root"></div>
+    <script src="/bundle.js"></script>
```

Comment:
Well, maybe, but initially I assumed it was ok to hardcode the `app`, `api` and `assets` namespaces, because I didn't really consider that any project would need to change these, but there's no harm in putting stuff like that in an env var.